### PR TITLE
Form 350/profile legal entity

### PR DESCRIPTION
**Pull request by matejpeluha**

### CODE REVIEW

File: next/components/forms/segments/UserProfile/UserProfileView.tsx [hunk 1]
```diff
@@ -82,8 +83,8 @@ const UserProfileView = () => {
           }}
           title={t('profile_detail.modal_title')}
         >
-          <AccountMarkdown content={t('profile_detail.modal_message')} variant="sm" />
-          <p className="mt-6">{t('profile_detail.modal_thanks')}</p>
+          <AccountMarkdown content={t('profile_detail.modal_message')} variant="sm" className={cx("text-center","md:text-left")} />
```
Comment (mpinter) [inline next/components/forms/segments/UserProfile/UserProfileView.tsx:86]: why adding cx, here and elsewhere - shouldn't it behave the same without it ?

**Review by mpinter — APPROVED** [review]
please let me know about the `cx` usage (and remove if it's not needed) - otherwise good to go

File: next/components/forms/segments/UserProfile/UserProfileView.tsx (hunk 1 above)
Comment (matejpeluha) [reply next/components/forms/segments/UserProfile/UserProfileView.tsx:86]: forgot to remove it while i tried some things

Comment (matejpeluha) [reply next/components/forms/segments/UserProfile/UserProfileView.tsx:86]: for better clean code. i always divide classNames to 2 separate params in cx. one are basic classes and second is media queries.

**Pull request merged**